src/activity-feed/ActivityFeedHeader.jsx

```diff
@@ -42,49 +42,29 @@ const HeaderActions = styled('div')`
   }
 `
 
-const Link = styled.a`
-  margin-bottom: 0;
-`
-
-export default class ActivityFeedHeader extends React.Component {
-  static propTypes = {
-    totalActivities: PropTypes.number,
-    contentText: PropTypes.string,
-    contentLink: PropTypes.string,
-  }
-
-  static defaultProps = {
-    totalActivities: 0,
-    contentText: null,
-    contentLink: null,
-  }
+const ActivityFeedHeader = ({ totalActivities, buttons }) => {
+  const headerText = totalActivities
+    ? pluralize('activity', totalActivities, true)
+    : 'Activities'
 
-  render() {
-    const { totalActivities, contentText, contentLink } = this.props
-    const headerText = totalActivities
-      ? pluralize('activity', totalActivities, true)
-      : 'Activities'
+  return (
+    <HeaderSummary>
+      <HeaderCount>
+        <H2>{headerText}</H2>
+      </HeaderCount>
+      <HeaderActions>{buttons && buttons}</HeaderActions>
```

Comment:
`HeaderActions` shouldn't render when buttons are empty.

```suggestion
      {buttons && <HeaderActions>{buttons}</HeaderActions>}
```